prime-router/src/main/resources/log4j2.xml

```diff
@@ -2,16 +2,7 @@
 <Configuration status="INFO">
     <Appenders>
         <Console name="Console" target="SYSTEM_OUT">
-            <PatternLayout>
-                <!-- Might be able to use entity references to avoid repeating this pattern over and over. -->
-                <LevelPatternSelector defaultPattern="%d{HH:mm:ss.SSS} [%t][%T] %-5level %logger{36} - %msg - %X%n">
-                  <!-- Add some color to ERROR and WARN -->
-                  <PatternMatch key="ERROR" pattern="%highlight{%d{HH:mm:ss.SSS} [%t][%T] %-5level %logger{36} - %msg}{bright,red}%n" />
-                  <PatternMatch key="WARN" pattern="%highlight{%d{HH:mm:ss.SSS} [%t][%T] %-5level %logger{36} - %msg}{bright,red}%n" />
-                  <!-- Also add a unique string that can trigger PagerDuty  -->
-                  <PatternMatch key="FATAL" pattern="%highlight{%d{HH:mm:ss.SSS} [%t][%T] FATAL-ALERT %logger{36} - %msg}{bright,red}%n" />
-                </LevelPatternSelector>
-           </PatternLayout>
+            <JsonTemplateLayout eventTemplateUri="classpath:jsonTemplate.json" maxStringLength="14000" />
```

Comment:
Does this layout require a max string length?  It also looks like it might be bigger for azure?
https://github.com/MicrosoftDocs/azure-docs/blob/main/includes/application-insights-limits.md

Reply:
`14000` seemed to be the length suggested here: https://learn.microsoft.com/en-us/azure/spring-apps/structured-app-log

I'm unsure if the same limits apply outside of a Spring app.